src/sonic-yang-models/yang-models/sonic-buffer-pool.yang

```diff
@@ -57,6 +61,17 @@ module sonic-buffer-pool {
                     type uint64;
                     description "Buffer Pool Xoff Threshold (in Bytes)";
                 }
+
+                leaf percentage {
+                    type uint8;
```

Comment:
Add range to percentage?

Reply:
same as the other PR. I do not prefer to so far.